[2025-08-19T09:07:39.427+0000] {task_context_logger.py:104} ERROR - Detected zombie job: {'full_filepath': '/opt/airflow/dags/data_audit_dag.py', 'processor_subdir': '/opt/airflow/dags', 'msg': "{'DAG Id': 'data_audit_dag', 'Task Id': 'data_pull_task', 'Run Id': 'scheduled__2025-08-19T08:00:00+00:00', 'Hostname': '37fc8edea13d', 'External Executor Id': '9084a680-5360-4c46-9542-a4c8bf59ce2d'}", 'simple_task_instance': <airflow.models.taskinstance.SimpleTaskInstance object at 0x74dd71e4b020>, 'is_failure_callback': True} (See https://airflow.apache.org/docs/apache-airflow/stable/core-concepts/tasks.html#zombie-undead-tasks)
